notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -220,6 +220,46 @@ father and son [Sherman Klump had a heart to heart talk with his father Cletus a
 giant hamster []
 
 
+movie: The Cell (2000)
+======================
+
+:: Title
+The Cell
+
+:: Description
+The Cell is a 2000 science fiction psychological horror film directed by
+Tarsem Singh in his directorial debut, and starring Jennifer Lopez, Vince
+Vaughn, and Vincent D'Onofrio.
+
+Synopsis: The film follows scientists as they use experimental technology to
```

Comment:
Just for the record: if you write "The film follows.." I don't mind either using or not using the initial "Synopsis". It is when you change vantage point to something like "The year is 2042 and..." that it gets weird otherwise. And also just for the record: you are switching to a notably different way of write your synopses here compared to what you did in other entries.

Reply:
I updated the synopsis text. But if you're wondering about the different styles, it's because I'm oftentimes just copy and pasting from Wikipedia, and that's how it is there.